### PR TITLE
Added email configuration settings to main Manifold configuration file.

### PR DESCRIPTION
**Pull request by ghost**
Email settings were added based on work done by Will Simpson, see:

    https://redmine.library.arizona.edu/issues/10827

Note that the settings are going in under the "manifold_api['env']"
data structure, rather than in the more specific keys under
"manifold_api['manifold_email_*']" because the more specific keys
don't seem to be recognized, at least as of v2.0.4.

Parameterization for the new configuration values is provided
in the variables file for the "openaz" group.

### CODE REVIEW

**Review by nazarenkoe — APPROVED** [review]
Looked at the code, don't know enough  about this particular file, but I don't see any glaring errors.

**Comment by mgsimpson-ua** [comment]
Used the feature branch to push changes to openaz-tst, which appears to have worked as intended; waiting on confirmation from Will before merging and pushing to -stg and -prd.

**Comment by simpsonw** [comment]
@mgsimpson-ua Worked for me as well, thanks!

**Pull request merged**
